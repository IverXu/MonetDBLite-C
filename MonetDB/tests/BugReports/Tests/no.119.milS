# >Number:         119
# >Category:       Module_kernel
# >Synopsis:       thetajoin gives incorrect output
# >Confidential:   no
# >Severity:       serious
# >Priority:       medium
# >Responsible:    boncz
# >State:          open
# >Class:          sw-bug
# >Submitter-Id:   unknown
# >Arrival-Date:   Wed Feb 11 11:26:18 1998
# >Originator:     niels
# >Organization:
# CWI
# >Release:        971007
# >Environment:
# All
# >Description:
# Simple script 
# a.insert(1,2);
# a.insert(2,1);
# a.thetajoin(a,LT).print;
# 2 1
# 2 2
# 1 1 
# 1 2

setoid(oid(20000000));
a := new(int,int);
a.insert(1,3);
a.insert(2,2);
a.insert(3,1);
a.thetajoin(a,LT).print;
a.thetajoin(a,LE).print;
a.thetajoin(a,EQ).print;
a.thetajoin(a,GE).print;
a.thetajoin(a,GT).print;
quit;
