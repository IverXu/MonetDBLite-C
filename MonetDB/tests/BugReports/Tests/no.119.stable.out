stdout of test 'no.119` in directory 'tests/BugReports` itself:

#> Mtest:	milS no.119 milS
#> Mtest:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db tests_BugReports -single no.119.milS
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:no.119.milS

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
# >Number:         119
# >Category:       Module_kernel
# >Synopsis:       thetajoin gives incorrect output
# >Confidential:   no
# >Severity:       serious
# >Priority:       medium
# >Responsible:    boncz
# >State:          open
# >Class:          sw-bug
# >Submitter-Id:   unknown
# >Arrival-Date:   Wed Feb 11 11:26:18 1998
# >Originator:     niels
# >Organization:
# CWI
# >Release:        971007
# >Environment:
# All
# >Description:
# Simple script 
# a.insert(1,2);
# a.insert(2,1);
# a.thetajoin(a,LT).print;
# 2 1
# 2 2
# 1 1 
# 1 2

setoid(oid(20000000));
a := new(int,int);
a.insert(1,3);
a.insert(2,2);
a.insert(3,1);
a.thetajoin(a,LT).print;
#-----------------#
# BAT:	  tmp_28  #
# (int)	  (int)	  #
#-----------------#
[ 2,	  1	  ]
[ 3,	  2	  ]
[ 3,	  1	  ]
a.thetajoin(a,LE).print;
#-----------------#
# BAT:	  tmp_28  #
# (int)	  (int)	  #
#-----------------#
[ 1,	  1	  ]
[ 2,	  2	  ]
[ 2,	  1	  ]
[ 3,	  3	  ]
[ 3,	  2	  ]
[ 3,	  1	  ]
a.thetajoin(a,EQ).print;
#-----------------#
# BAT:	  tmp_28  #
# (int)	  (int)	  #
#-----------------#
[ 1,	  1	  ]
[ 2,	  2	  ]
[ 3,	  3	  ]
a.thetajoin(a,GE).print;
#-----------------#
# BAT:	  tmp_28  #
# (int)	  (int)	  #
#-----------------#
[ 1,	  3	  ]
[ 1,	  2	  ]
[ 1,	  1	  ]
[ 2,	  3	  ]
[ 2,	  2	  ]
[ 3,	  3	  ]
a.thetajoin(a,GT).print;
#-----------------#
# BAT:	  tmp_28  #
# (int)	  (int)	  #
#-----------------#
[ 1,	  3	  ]
[ 1,	  2	  ]
[ 2,	  3	  ]
quit;
